Specification/5._Application_level_conventions.md

```diff
@@ -8,33 +8,31 @@ permalink: /Specification/5._Application_level_conventions/
 
 ### Message type ID
 
-As specified in the CAN bus transport layer specification, the valid range for message type ID is from 0 to 65535,
-inclusive.
+The valid range for *message type ID* is from 0 to 65535, inclusive (specified in [CAN bus transport layer](/Specification/4._CAN_bus_transport_layer)).
 The range is segmented as follows:
 
 ID range        | Purpose
 ----------------|--------------------------------------------------------------
-[0, 20000)      | Standard message types
-[20000, 21000)  | Vendor-specific message types
-[21000, 65536)  | Reserved for future use
```

Comment:
@pavel-kirienko The anonymous message types not shown here - range 0 to 3 I think. Should they be?

Reply:
@pavel-kirienko You missed response to this comment.

Reply:
Sorry. Anonymous messages are standard messages, so there is no need to dedicate a separate section to them.

Reply:
Thanks! Good to see this merged. 
FYI @pavel-kirienko I am unlikely to get much/anything done next 2 weeks. On holiday.